pkgs/by-name/it/itgmania/package.nix

```
@@ -0,0 +1,92 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  git,
+  nasm,
+  alsa-lib,
+  glew,
+  glib,
+  gtk3,
+  libmad,
+  libogg,
+  libpulseaudio,
+  libusb-compat-0_1,
+  libvorbis,
+  pkg-config,
+  udev,
+  xorg,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "itgmania";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "itgmania";
+    repo = "itgmania";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-SAEYkAPNUjGNfNnHfwyOj65i2SpEX0ct/fREob5/6fI=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    nasm
+  ];
+
+  buildInputs = [
+    alsa-lib
+    git
+    glew
+    glib
+    gtk3
+    libmad
+    libogg
+    libpulseaudio
+    libusb-compat-0_1
+    libvorbis
+    pkg-config
+    udev
+    xorg.libXtst
```

you should be able to import `libXtst` directly
```suggestion
    libXtst
```